cmd/wallet-sdk-gomobile/docs/packages.md

```diff
@@ -0,0 +1,400 @@
+</tr>
+<tr>
+<td> 
+
+```
+package api
+
+type CredentialReader interface {
+	Get(...) (*JSONObject, error)
+	GetAll() (*JSONArray, error)
+}
+```
+
+</td>
+<td>
+
+```kotlin
+import dev.trustbloc.wallet.sdk.api
+
+api.CredentialReader
+```
+
+</td>
+<td>
+
+```swift
+import Walletsdk
+
+ApiCredentialReader
+```
+
+</td>
+</tr>
+</table>
+
+### Credential Querying
+
+<table>
+<tr>
+<td> Go Source Code </td> <td> Java (as called from Kotlin) </td> <td> Obj-C (as called from Swift) </td>
+</tr>
+<tr>
+<td> 
+
+```
+package credential
+
+func NewInquirer(
+	...
+) *Inquirer {
+	...
+}
+```
+
+</td>
+<td>
+
+```kotlin
+import dev.trustbloc.wallet.sdk.credential
+
+val inquirer = credential.newInquirer(...)
+
+inquirer.query(...)
+```
+
+</td>
+<td>
+
+```swift
+import Walletsdk
+
+let inquirer = CredentialNewInquirer(...)
+
+inquirer.query(...)
+```
+
+</td>
+</tr>
+</table>
+
+### DID Creator and Resolver
+
+<table>
+<tr>
+<td> Go Source Code </td> <td> Java (as called from Kotlin) </td> <td> Obj-C (as called from Swift) </td>
+</tr>
+<tr>
+<td> 
+
+```
+package did
+
+func NewCreatorWithKeyWriter(
+	...
+) (*Creator, error) {
+    ...
+}
+
+func NewResolver() *Resolver {
+	...
+}
+
+```
+
+</td>
+<td>
+
+```kotlin
+import dev.trustbloc.wallet.sdk.did
+
+did.newCreatorWithKeyWriter(...)
+did.newResolver()
+```
+
+</td>
+<td>
+
+```swift
+import Walletsdk
+
+DidNewCreatorWithKeyWriter(...)
+DidNewResolver()
+```
+
+</td>
+</tr>
+</table>
+
+### In-Memory Credential DB
+
+<table>
+<tr>
+<td> Go Source Code </td> <td> Java (as called from Kotlin) </td> <td> Obj-C (as called from Swift) </td>
+</tr>
+<tr>
+<td> 
+
+```
+package credential
+
+func NewInMemoryDB() *DB {
+	...
+}
+```
+
+</td>
+<td>
+
+```kotlin
+import dev.trustbloc.wallet.sdk.credential
+
+credential.newInMemoryDB()
+```
+
+</td>
+<td>
+
+```swift
+import Walletsdk
+
+CredentialNewInMemoryDB()
+```
+
+</td>
+</tr>
+</table>
+
+### Local KMS
+
+<table>
+<tr>
+<td> Go Source Code </td> <td> Java (as called from Kotlin) </td> <td> Obj-C (as called from Swift) </td>
+</tr>
+<tr>
+<td> 
+
+```
+package localkms
+
+func NewKMS(...) (*KMS, error) {
+	...
+}
+```
+
+</td>
+<td>
+
+```kotlin
+import dev.trustbloc.wallet.sdk.localkms
+
+localkms.newKMS(...)
+```
+
+</td>
+<td>
+
+```swift
+import Walletsdk
+
+LocalkmsNewKMS(...)
+```
+
+</td>
+</tr>
+</table>
+
+### Linked Domains
+
+<table>
+<tr>
+<td> Go Source Code </td> <td> Java (as called from Kotlin) </td> <td> Obj-C (as called from Swift) </td>
+</tr>
+<tr>
+<td> 
+
+```
+package ld
+
+func NewDocLoader() *DocLoader {
+	...
+}
+```
+
+</td>
+<td>
+
+```kotlin
+import dev.trustbloc.wallet.sdk.ld
+
+ld.newDocLoader()
+```
+
+</td>
+<td>
+
+```swift
+import Walletsdk
+
+LdNewDocLoader()
+```
+
+</td>
+</tr>
+</table>
+
+### OpenID4CI
+
+<table>
+<tr>
+<td> Go Source Code </td> <td> Java (as called from Kotlin) </td> <td> Obj-C (as called from Swift) </td>
+</tr>
+<tr>
+<td> 
+
+```
+package openid4ci
```

Comment:
I used to have the standalone credential display resolver in a separate package. I moved it into the same package as the interaction since it's specific to `openid4ci` anyway. It has the side benefit of resolving the word duplication issues and reduces the number of Android packages by 1.